lib/components/Buttons/styles.ts

```diff
@@ -37,38 +37,15 @@ export const button_primary = css`
 `;
 export const button_secondary = css`
   ${button};
-  display: flex;
-  padding: 0rem 2.4rem;
-  align-items: center;
-  justify-content: center;
-  appearance: none;
-  text-decoration: none;
-  /* min-height to conform with Lighthouse min tap-target */
-  min-height: 4.8rem;
-  cursor: pointer;
+  ${button_primary};
   border: 1px solid var(--klima-green);
   border-radius: 0.4rem;
-  transition: opacity 0.3s ease 0s;
   background-color: transparent;
-
-  &:active:not(:disabled) {
-    transform: scale(0.95);
-    opacity: 0.6;
-  }
   &,
   &:hover:not(:disabled),
   &:visited {
     color: var(--klima-green); /* same in darkmode */
   }
-
-  &:disabled {
-    border-color: var(--surface-03);
-    cursor: not-allowed;
-  }
-
-  &.icon {
-    gap: 0.8rem;
-  }
```

Comment:
In this file I would make a clear distinction between base, Primary and Secondary

```TS
const buttonBase = css`
  ${button};
  display: flex;
  padding: 0rem 2.4rem;
  align-items: center;
  justify-content: center;
  appearance: none;
  text-decoration: none;
  /* min-height to conform with Lighthouse min tap-target */
  min-height: 4.8rem;
  cursor: pointer;
  border-radius: 0.4rem;
  transition: opacity 0.3s ease 0s;
`;

export const buttonPrimary = css`
  ${buttonBase};
  background-color: var(--klima-green);

  &:active:not(:disabled) {
    transform: scale(0.95);
    opacity: 0.6;
  }
  &,
  &:hover:not(:disabled),
  &:visited {
    color: white; /* same in darkmode */
  }

  &:disabled {
    background-color: var(--surface-03);
    cursor: not-allowed;
  }

  &.icon {
    gap: 0.8rem;
  }

  &.gray {
    background-color: var(--surface-01);
    color: var(--font-02);

    &,
    &:hover,
    &:hover:not(:disabled),
    &:visited {
      color: var(--font-02); /* same in darkmode */
    }
  }

  &.blueRounded {
    background-color: var(--klima-blue);
    color: var(--surface-01);
    border-radius: 0.8rem;
    &,
    &:hover,
    &:hover:not(:disabled),
    &:visited {
      color: var(--surface-01); /* same in darkmode */
    }
  }
`;
export const buttonSecondary = css`
  ${buttonBase}
  border: 1px solid var(--klima-green);
  border-radius: 0.4rem;
  background-color: transparent;
  &,
  &:hover:not(:disabled),
  &:visited {
    color: var(--klima-green); /* same in darkmode */
  }

  &.gray {
    border-color: var(--surface-01);
    color: var(--surface-01);

    &,
    &:hover,
    &:hover:not(:disabled),
    &:visited {
      color: var(--font-01); /* same in darkmode */
    }
  }

  &.blueRounded {
    border-color: var(--klima-blue);
    color: var(--klima-blue);
    border-radius: 0.8rem;
    &,
    &:hover,
    &:hover:not(:disabled),
    &:visited {
      color: var(--klima-blue); /* same in darkmode */
    }
  }
`;
```